test/systemtests/network_test.go

```diff
@@ -338,6 +338,7 @@ func (s *systemtestSuite) testNetworkAddDeleteTenant(c *C, encap, fwdmode string
 			}
 		}
 
+		time.Sleep(3 * time.Second)
```

Comment:
what is this timing issue and why it will go away if we remove this 3 seconds wait ?